Update clspv reflection non-semantic instruction set (#220)
KhronosGroup/SPIRV-Registry: nonsemantic/NonSemantic.ClspvReflection.asciidoc

'BufferSize' |
|=====

[cols="5"]
|=====
5+|[[NormalizedSamplerMaskPushConstant]]*NormalizedSamplerMaskPushConstant* +
 +
 Missing before *version 6*. +
 +
Declares a *PushConstant* entry to specify the sampler mask of a non literal

> sampler. It means that the kernel is accessing a 3D images with this sampler.
> But as accessing 3D images with a sampler using unnormalized coordinates is not
> valid, *clspv* will have generated the code to normalized the coordinates.

Reviewer: ```suggestion
sampler. It means that the kernel accesses 3D images with this sampler,
but Vulkan does not allow accessing 3D images with a sampler using unnormalized coordinates. *Clspv* will generates code to normalized the coordinates.
```

> *clspv* will choose at runtime whether to use the original coordinates or the

Reviewer: ```suggestion
*Clspv* will choose at runtime whether to use the original coordinates or the
```
